foliant/backends/mkdocs.py

```diff
@@ -15,7 +18,15 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self._mkdocs_config = self.config.get('backend_config', {}).get('mkdocs', {})
-        self._mkdocs_site_dir_name = f'{self.get_slug()}.mkdocs'
+
+        overridden_slug = self._mkdocs_config.get('slug', None)
```

Comment:
The entire green section can be replaced with:

`self._mkdocs_site_dir_name = f'{self._mkdocs_config.get('slug', self.get_slug()}'`

Reply:
Replaced with:

`self._mkdocs_site_dir_name = f'{self._mkdocs_config.get("slug", self.get_slug())}.mkdocs'`